server/service/osquery.go

```diff
@@ -1010,6 +1010,8 @@ func (svc *Service) SubmitDistributedQueryResults(
 			logging.WithErr(ctx, err)
 		}
 
+		// NOTE: if the installers for the policies here are not scoped to the host via labels, we update the policy status here to stop it from showing up as "failed" in the
+		// host details.
```

Comment:
Just to double-check, we opted to still send the policies but filter-them out here instead of not sending those policies in the first place, right? That may be ok, just want to make sure this is deliberate (e.g. it means the host may receive policy results while not in scope for the labels, but then it becomes in scope due to membership changes, the policy won't be reported as "newly-failing" in that case, right? It would be reported as "still failing"? Is that ok?)